newIDE/app/src/PropertiesEditor/index.js

```diff
@@ -608,23 +605,21 @@ const PropertiesEditor = ({
         return (
           <div key={field.name}>
             <Subheader>{field.name}</Subheader>
-            <div style={styles.subPropertiesEditorContainer}>
```

Comment:
Why remove the margin left for sub properties?

Reply:
It didn't really add clarity by indenting them, removing it looks cleaner and more aligned with the rest